packages/backend-function/src/function_client_config_generator.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import path from 'path';
+
+const lambdaConfigTemplate = (
+  functionName: string,
+  modelIntrospectionSchema: string
+) => `// This file is auto-generated by Amplify. Edits will be overwritten.
+import { env } from "../env/${functionName}";
+
+export const libraryOptions = {
```

Comment:
I seem to have lost a comment from @sobolk suggesting this content be pushed down into a file to be imported.

Which package would this file be in?  I had been advocating for amplify-js, however, that doesn't seem like its going to be possible.

Since this is essentially customer code, could this behavior be usable/consumable from the backend package? It seems safe to assume this will be present during the build.

Reply:
It does not have to be "imported" in literal sense. See https://github.com/aws-amplify/amplify-backend/blob/9a5d8e3f985e7b4ecf46babe709ca482d1d03b4f/packages/backend-function/src/factory.ts#L411-L419 .

The point is to get it under compiler and be able to write unit tests for it.